NEWS.md

```diff
@@ -20,6 +20,26 @@
   `KernelModulesExclude=` or `KernelModulesInitrdExclude=` are used.
 - Added `ToolsTreeDistribution=` match.
 - Removed `vmspawn` verb and replaced it with `VirtualMachineMonitor=`.
+- New specifiers for various directories were added (`%D`, `%P`, `%C`)
+- Added `ForwardJournal=` to have systemd inside a container/VM forward
+  its journal to the specified file or directory.
+- Systemd scopes are now allocated for qemu, swtpm and virtiofsd if
```

Comment:
```suggestion
- Systemd scopes are now allocated for qemu, swtpm, virtiofsd, and systemd-journal-remote if
```
Thought the NEWS commit might need to be ordered after the journal-remote commit ;)